packages/editor/src/client/routes/articleList.tsx

```diff
@@ -122,6 +136,10 @@ export function ArticleList() {
           data={articles}
           sortColumn={sortField}
           sortType={sortOrder}
+          rowClassName={rowData => {
+            if (rowData && rowData.id === duplicatedDataId) return 'duplicated-row'
```

Comment:
Another way to write this would be:
`return rowData?.id === duplicatedDataId ? 'duplicated-row' : ''`